tests/foreman/api/test_architecture.py

```diff
@@ -53,7 +53,8 @@ def test_positive_create_with_name(self):
 
         :CaseImportance: Critical
         """
-        for name in valid_data_list():
+        for key in valid_data_list().keys():
+            name = valid_data_list()[key]
```

Comment:
Why? Did the structure itself change in some other PR?

Reply:
Not sure. What I can say, that we are now, in 6.8 working incorrectly with valid_data_list(), you can look results in report portal. 

But yes I would like to know too what happened and why now is necessary to use dict instead of list.

Reply:
but as I look at it today, better solution is to use only values.
`for name in valid_data_list().values():`

Reply:
OK, let's not dig too deep and just fix it.
I agree `.values()` looks better.

Reply:
this was discussed yesterday in gchat, the reasoning is this change:

https://github.com/SatelliteQE/robottelo/pull/7824/files#diff-ed4419a639eb031bb8fe2b4f9100b28fL50

Reply:
Do you want to change it to a more readable format using `.values` before merging?